lib/ocpp/v16/charge_point_impl.cpp

```diff
@@ -2010,6 +2014,11 @@ void ChargePointImpl::handleTriggerMessageRequest(ocpp::Call<TriggerMessageReque
         valid = false;
     }
 
+    if (this->connection_state = ChargePointConnectionState::Pending) {
+        EVLOG_debug << "Set connection_state from pending to accepted";
+        this->connection_state = ChargePointConnectionState::Booted;
+    }
```

Comment:
How do you come to this conclusion? From my understanding only the BootNotification.Conf should change this.